src/collectd.conf.pod

```diff
@@ -330,7 +330,7 @@ hostname will be determined using the L<gethostname(2)> system call.
 
 If B<Hostname> is determined automatically this setting controls whether or not
 the daemon should try to figure out the "fully qualified domain name", FQDN.
-This is done using a lookup of the name returned by C<gethostname>. This option
+This is done using the value of struct addrinfo->ai_canonname returned by  C<getaddrinfo()>. This option
```

Comment:
I'd believe, the target audience for the config file won't understand, what this means exactly. Would you be able to rephrase? What does a struct addrinfo ... mean for the end user (that is probably someone using collectd to monitor something).

Reply:
I have rephrased it & also enhanced it to make clear it uses the first FQDN it can get from a network interface. I used the term "web address" as it's hard for me to describe behaviour without  repeating the previous sentence that it looks up the FQDN.